sdk/containerregistry/Microsoft.Azure.ContainerRegistry/tests/Tests/ScopeTests.cs

```diff
@@ -104,42 +104,6 @@ private void VerifyManifest(Manifest baseManifest, Manifest actualManifest)
                 Assert.Equal(baseManifestV2.Config.MediaType, actualManifestV2.Config.MediaType);
                 Assert.Equal(baseManifestV2.Config.Size, actualManifestV2.Config.Size);
             }
-            if (baseManifest.GetType() == typeof(V1Manifest))
```

Comment:
Why is this removed?

Reply:
I am only using V2Manifest in the tests.